features/bevragen/autorisatie-gba.feature

```diff
@@ -294,7 +283,7 @@ Functionaliteit: autorisatie voor het gebruik van de API
       | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
       | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
       | status   | 403                                                                     |
-      | code     | authorization                                                           |
+      | code     | unauthorizedField                                                       |
       | instance | /haalcentraal/api/brp/personen                                          |
 
     @fout-case
```

Comment:
Is het hier relevant of er personen wel/niet wordt gevonden met de zoekvraag? De check of de afnemer geautoriseerd is voor de gevraagde fields gebeurt toch voor het zoeken van personen?

Reply:
nee, niet. om dat te illustreren heb ik juist expres een scenario gemaakt waar het veld waar geen autorisatie voor is geen waarde heeft. dan moet er toch een fout komen. voor testen had dat niet uitgemaakt en is alle Gegeven op de persoon irrelevant